lib/galaxy/tools/__init__.py

```diff
@@ -131,6 +131,7 @@
     "CONVERTER_maf_to_fasta_0",
     "CONVERTER_maf_to_interval_0",
     "CONVERTER_wiggle_to_interval_0",
+    "CONVERTER_bed_to_fli_0",
```

Comment:
I think those are mostly alphabetic, could you move this line just below "# Converters" ?